src/presentation/settings/menu-settings.tsx

```diff
@@ -16,7 +17,11 @@ const SettingsMenuSettings: React.FC = () => {
   const handleDeepRestorer = () => history.push(SETTINGS_DEEP_RESTORER_ROUTE);
 
   return (
-    <ShellPopUp className="h-popupContent" currentPage="Settings">
+    <ShellPopUp
+      className="h-popupContent"
+      currentPage="Settings"
+      backBtnCb={() => history.push(DEFAULT_ROUTE)}
```

Comment:
why not `goBack`? this will add a new route to the stack

Reply:
to be consistent with the others screens (we also push to home in `Send` and `Receive`). We use  `goback` in all sub-menus (Settings for instance).